crashlytics/Shared/ContentView.swift

```diff
@@ -25,6 +45,17 @@ struct ContentView: View {
         Text("Crash")
       }
       .navigationTitle("Crashlytics Example")
+      .onAppear {
```

Comment:
Have you considered using the `task` modifier?

Reply:
Yes using the `.task` modifier is a good suggestion. I will make the change in a following PR.